include/SVF-FE/GEPTypeBridgeIterator.h

```diff
@@ -72,8 +72,16 @@ class generic_bridge_gep_type_iterator : public std::iterator<std::forward_itera
         if ( CurTy.getInt() )
             return CurTy.getPointer();
 #if LLVM_VERSION_MAJOR >= 11
-        Type *CT = CurTy.getPointer();
-        return CT;
+        Type * CT = CurTy.getPointer();
+        auto * ST = dyn_cast<StructType>(CT);
+        auto Array = dyn_cast<ArrayType>(CT);
+        auto Vector = dyn_cast<VectorType>(CT);
+        if (ST)
+            return ST->getTypeAtIndex(getOperand());
+        else if (Array)
+            return Array->getElementType();
+        else if (Vector)
+            return Vector->getElementType();
```

Comment:
do we need a `else` branch here?

Reply:
The old CompositeType support three type (ArrayType, StructType and VectorType), so I think maybe the else branch is unreachable?

Reply:
Anyway I can add a else branch such as `return CT`.

Reply:
then please add an assertion in the else branch.